lib/src/client.dart

```diff
@@ -1161,10 +1161,11 @@ class Client extends MatrixApi {
     _archivedRooms.add(ArchivedRoom(room: archivedRoom, timeline: timeline));
   }
 
-  GetVersionsResponse? _versionsCache;
+  final _versionsCache =
+      AsyncCache<GetVersionsResponse>(const Duration(hours: 1));
```

Comment:
this needs to be reset on logout, I don't really see that happening anywhere?

Reply:
good catch done